src/graphql/resolvers/common.ts

```diff
@@ -178,13 +232,18 @@ export const resolveNumberFilter = (value, propertyName) => {
   return filter;
 };
 
-export const getMongoSortDirection = value => (value === 'ASCENDING' ? 1 : -1);
+export const getMongoSortDirection = (value: string) => (value === 'ASCENDING' ? 1 : -1);
```

Comment:
If I recall correctly this is matching a GraphQL enum that can be either ASCENDING or DESCENDING. `string` is fine for the type, but something you can do with Typescript if you're working with string enums is define a type as a union of string literals. You could make a type like the following to use here instead:
`type SortDirection = 'ASCENDING' | 'DESCENDING';`